scripts/archive.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+PACKAGE_NAME=${TOOL_NAME:?}-${VERSION:?}
```

Comment:
I'm not sure I've ever seen release artefacts with versions on them on github, as they are already uploaded to a specific version. Do you have any examples?
It would require anyone downloading them from a script to dynamically handle the version in the filename (I know they would already be doing it for the url)

Reply:
You're right, but once you have downloaded from release page, you can't tell which one is which version. It's better for me to have version number in the extracted directory name.

<img width="256" alt="screen shot 2018-02-27 at 8 26 13" src="https://user-images.githubusercontent.com/6007952/36701763-e31524dc-1b97-11e8-95bd-0a7556cfb0da.png">

By the way I didn't change the zipfile name. `xcodegen.zip`

It's easy to handle the extracted directory name to execute `install.sh`. Like this.
```
curl https://latest.release.zip -o tmp/a.zip
(cd tmp; unzip a.zip; ./**/install.sh)
```

> (I know they would already be doing it for the url)

They wouldn't have to, you know? If they use GitHub API to get the latest release.

Reply:
I see, it’s not the zip filename, it’s the unzipped directory name. I would say that’s even worse in my opinion, as CI script authors would assume the directory would have the same name as the zip file. Forcing them to use a glob pattern or specify a version path isn’t great. (Especially if they use a GitHub api as you say from a high level and then don’t think about the version afterwards)
I think your use case of wanting to know which file you unzipped is an edge case and comes about just because you have so many duplicates from testing.
Thoughts?

Reply:
Indeed, you're right. I'll change the unzipped dir name to `xcodegen` as well 👌

Reply:
Done in 6cd4367